app/javascript/components/upload/UploadWizard.jsx

```diff
@@ -357,67 +398,85 @@ export function RawUploadWizard({ studyAccession, name }) {
   useEffect(() => {
     fetchStudyFileInfo(studyAccession).then(response => {
       response.files.forEach(file => formatFileFromServer(file))
+      setIsAnnDataExperience(response.files?.filter(AnnDataFileFilter)?.length > 0 && response.feature_flags?.ingest_anndata_file)
       setServerState(response)
       setFormState(_cloneDeep(response))
       setTimeout(pollServerState, POLLING_INTERVAL)
     })
+
     window.document.title = `Upload - Single Cell Portal`
   }, [studyAccession])
 
   const nextStep = STEPS[currentStepIndex + 1]
   const prevStep = STEPS[currentStepIndex - 1]
+
+  /**
+   * Returns the appropriate content to display for the UploadWizard
+   * @param  formState
```

Comment:
Formal docstring entries for parameters are often helpful, but only if they convey more than what the function signature does in the code -- e.g. a brief note on what `formState` is and perhaps (as it's not TypeScript) what the type of `formState` is.

Let's either add a rough type annotation and/or brief description, or remove this line as it's duplicative with the signature.  (I think the latter would be fine, though I'd certainly support the former.)

```suggestion
```